bike_rack/parse_utils.py

```diff
@@ -1,6 +1,34 @@
 from PyPDF2 import PdfFileReader
 import datetime as dt
 
+REPLACEMENTS = [
```

Comment:
I feel like this PR should go ahead and remove `replace_chars()` from `utils.py` because the method `Minutes.parse_pages()` takes over that functionality

Reply:
Sounds good, I can take care of this in the broader refactoring of `utils.py`